PivotGrid: Update Field Chooser demo descriptions (#998)
DevExpress/devextreme-demos: JSDemos/Demos/PivotGrid/StandaloneFieldChooser/description.md

> Along with a field chooser [integrated into the PivotGrid](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/), DevExtreme includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component. Unlike its integrated counterpart, the standalone field chooser can be displayed on a page permanently.

Reviewer: "Displayed permanently" sounds a bit weird to me. Partly because of passive voice, partly because the word "permanently" doesn't seem right. 

```suggestion
Along with a field chooser [integrated into the PivotGrid](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/), DevExtreme includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component. Unlike its integrated counterpart, the standalone field chooser can remain visible on the page at all times.
```

> Along with a field chooser [integrated into the PivotGrid](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/), DevExtreme includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component. Unlike its integrated counterpart, the standalone field chooser can remain visible on the page at all times.

Reviewer 2: ```suggestion
DevExtreme also includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component with the [integrated](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/) field chooser. Unlike its integrated counterpart, the standalone field chooser can remain visible on the page at all times.
```

Author: > a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component with the [integrated](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/) field chooser

The standalone field chooser doesn't include the integrated field chooser. It's DevExtreme that includes a standalone field chooser and a field chooser integrated into the PivotGrid.

Reviewer 2: Why not just say 'DevExtreme also includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component'?

Author: We have two different demos for the integrated and standalone field choosers. Those who enter our website from the standalone field chooser demo will be confused by this "also". I quess the following version would work:

```suggestion
DevExtreme includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component. Unlike its [integrated counterpart](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/), the standalone field chooser can remain visible on the page at all times.
```

Reviewer 2: Ok

Follow the steps below to connect the PivotGridFieldChooser with the PivotGrid:

1. **Bind both components to the same data source**         
Assign the same [PivotGridDataSource](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/) instance to the [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#dataSource) property of both components.

1. *(Optional)* **Disable the integrated field chooser**            
Both field choosers can work simultaneously, but if you want to disable the integrated version, set the PivotGrid's [fieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/).[enabled](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/#enabled) property to **false**.

1. *(Optional)* **Apply field changes on demand**           

> When users move fields within the field chooser, the changes are applied to the connected PivotGrid instantly. To change this behavior, you can set the [applyChangesMode](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#applyChangesMode) to *"onDemand"*. In this mode, the changes are applied/canceled only when you call the [applyChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#applyChanges)/[cancelChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#cancelChanges) method or set a new [state](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#state). You can add a UI for these methods. For example, in this demo, they are bound to the Apply and Cancel buttons. Select "onDemand" from the Apply Changes Mode drop-down menu to display these buttons.

Reviewer 3: >You can add a UI for these methods.

Not sure that this sentence is valid. I would rephrase it as follows: 'You can call these methods from the UI' or something similar. 

However, if you are sure that this sentence is OK, just skip this comment.

Reviewer: ```suggestion
When users move fields within the field chooser, the changes are applied to the connected PivotGrid instantly. To change this behavior, you can set the [applyChangesMode](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#applyChangesMode) to *"onDemand"*. In this mode, the changes are applied/canceled only when you call the [applyChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#applyChanges)/[cancelChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#cancelChanges) method or set a new [state](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#state). This demo adds custom Apply and Cancel buttons that call the methods mentioned above. Select "onDemand" from the Apply Changes Mode drop-down menu to display these buttons.
```

> When users move fields within the field chooser, the changes are applied to the connected PivotGrid instantly. To change this behavior, you can set the [applyChangesMode](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#applyChangesMode) to *"onDemand"*. In this mode, the changes are applied/canceled only when you call the [applyChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#applyChanges)/[cancelChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#cancelChanges) method or set a new [state](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#state). You can add a UI for these methods. For example, in this demo, they are bound to the Apply and Cancel buttons. Select "onDemand" from the Apply Changes Mode drop-down menu to display these buttons.

Reviewer 2: ```suggestion
When users move fields within the field chooser, the changes are applied to the connected PivotGrid instantly. To change this behavior, set the [applyChangesMode](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#applyChangesMode) to *"onDemand"*. In this mode, the changes are applied/canceled only when you call the [applyChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#applyChanges)/[cancelChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#cancelChanges) method or set a new [state](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#state). You can add a UI for these methods. For example, in this demo, they are bound to the Apply and Cancel buttons. Select "onDemand" from the Apply Changes Mode drop-down menu to display these buttons.
```

> The standalone and integrated field choosers share features. For example, both field choosers include three [layouts](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#layout). You can spot the difference between them in this demo. For information about other available features, refer to the [Integrated Field Chooser](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/) demo.

Reviewer: No explicit information on how to "spot differences". 

```suggestion
The standalone and integrated field choosers share features. For example, both field choosers include three [layouts](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#layout). You can use the Radio Buttons to switch between them and review differences. For information about other available features, refer to the [Integrated Field Chooser](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/) demo.
```

> The standalone and integrated field choosers share features. For example, both field choosers include three [layouts](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#layout). You can use the radio buttons to switch between them and review differences. For information about other available features, refer to the [Integrated Field Chooser](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/) demo.

Reviewer 2: ```suggestion
The standalone and integrated field choosers share features. For example, both field choosers include three [layouts](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#layout). You can use the radio buttons to switch between field choosers and review the differences. For information about other available features, refer to the [Integrated Field Chooser](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/) demo.
```
